Fabric.Terminology.API/Bootstrapping/Middleware/AuthorizationMiddleware.cs

```diff
@@ -0,0 +1,41 @@
+#pragma warning disable CA1309 // Use ordinal stringcomparison
+#pragma warning disable SA1503 // Braces must not be omitted
+namespace Fabric.Terminology.API.Bootstrapping.Middleware
+{
+    using System.Linq;
+    using System.Threading.Tasks;
+
+    using LibOwin;
+    using AppFunc = System.Func<System.Collections.Generic.IDictionary<string, object>, System.Threading.Tasks.Task>;
+
+    public sealed class AuthorizationMiddleware
+    {
+        public static AppFunc Inject(AppFunc next, string[] requiredScopes, string[] allowedPaths = null) =>
+            env =>
+                {
+                    var ctx = new OwinContext(env);
+
+                    if (ctx.Request.Method == "OPTIONS") return next(env);
```

Comment:
I dont get why a "OPTIONS" request would have a return here?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/OPTIONS

Reply:
@benjanderson  `AuthorizationMiddleware` is actually a Fabric team class which I copied in to make it easier to step through while I got the Identity/Auth integration sorted.  Once we have the final permissions, and I make the changes, my plan is to drop this class and pull in the [Fabric.Platform.Auth](https://www.nuget.org/packages/Fabric.Platform.Auth/) NuGet package.